src/app/view/modules/object/object.component.ts

```diff
@@ -97,4 +104,39 @@ export class ObjectComponent implements OnChanges, OnInit {
     }
```

Comment:
Yes, this has been on my to-do list for a while now, i planned to make separate a PR which refactors/fixes this and other rough spots in the code of `ObjectComponent`. 
There is also the issue that currently the initial resource is requested twice (`ngOnchanges` and `ngOnINit`), and the `resource` property could be named less confusingly since it is not a resource but a sequence of resources.